tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,77 @@
+import os
+from threading import Thread
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import requests
+import datetime
+
+from todo_app.app_config import Config
+from todo_app.app import create_app
+
+def create_trello_board():
+    config = Config()
+    response = requests.post(
+        url=f'{config.TRELLO_BASE_URL}/boards',
+        params={
+            'key': config.TRELLO_API_KEY,
+            'token': config.TRELLO_TOKEN,
+            'name': f'{datetime.datetime.now().strftime("%Y%m%d%H%M%S")}_Selenium_Test_Board'
+        }
+    )
+
+    return response.json()['id']
+
+def delete_trello_board(board_id):
+    config = Config()
+    response = requests.delete(
+        url=f'{config.TRELLO_BASE_URL}/boards/{board_id}',
+        params={
+            'key': config.TRELLO_API_KEY,
+            'token': config.TRELLO_TOKEN,
+        }
+    )
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    board_id = create_trello_board()
+    os.environ['TRELLO_BOARD_ID'] = board_id
+
+    # construct the new application
+    application = create_app()
+
+    # start the app in its own thread
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+
+    # Tear Down
+    thread.join(1)
+    delete_trello_board(board_id)
+
+@pytest.fixture(scope='module')
+def driver():
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    opts.add_argument('--disable-dev-shm-usage')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
```suggestion
    with webdriver.Chrome(options=opts) as driver:
```

If you skip specifying a file location, it will look for the chromedriver executable in the current folder **or** on your PATH. Which is preferable in case people have multiple projects using chromedriver.

Given that your chromedriver is in this folder, you may want to add it to your .gitignore file to avoid accidentally committing it.